src/components/academy/grading/index.tsx

```diff
@@ -181,6 +199,68 @@ class Grading extends React.Component<IGradingProps, State> {
               onChange={this.handleGroupsFilter}
             />
           </div>
+
```

Comment:
I understand where your coming from but isnt it better to use the filter within the table itself? The checkboxes are all doing the same thing (even if its more intuitive from a UI standpoint)